etc/ci/performance/submit_to_perfherder.py

```diff
@@ -0,0 +1,380 @@
+#!/usr/bin/env python3
+
+import argparse
+from functools import partial, reduce
+import json
+import operator
+import os
+import random
+import string
+from thclient import (TreeherderClient, TreeherderResultSetCollection,
+                      TreeherderJobCollection)
+import time
+
+from runner import format_result_summary
+
+
+def geometric_mean(iterable):
+        filtered = list(filter(lambda x: x > 0, iterable))
+        return (reduce(operator.mul, filtered)) ** (1.0 / len(filtered))
+
+
+def format_testcase_name(name):
+    temp = name.replace('http://localhost:8000/page_load_test/', '')
+    temp = temp.replace('http://localhost:8000/tp6/', '')
```

Comment:
tp6 is my experimental test suite, which I haven't turn on by default. TP5 is from 2011, I think we can build a TP6 for 2016, but not ready yet.